nap/engine.py

```diff
@@ -74,7 +75,7 @@ def _generate_url(self, url_type='lookup', resource_obj=None, **kwargs):
             }
 
             url_match_vars = dict([
-                (k, v) for (k, v) in model_keywords.items()
+                (k, v) for (k, v) in list(model_keywords.items())
```

Comment:
Is there a performance improvement forcing this from an iterator to a list in a comprehension?